lib/dirac_staggered.cpp

```diff
@@ -126,10 +127,21 @@ namespace quda {
     // do nothing
   }
 
+  /* Creates the coarse grid dirac operator
+  Takes: multigrid transfer class, which knows
+  about the coarse grid blocking, as well as
+  having prolongate and restrict member functions
+  
+  Returns: Color matrices Y[0..2*dim] corresponding
+  to the coarse grid operator.  The first 2*dim
+  matrices correspond to the forward/backward
+  hopping terms on the coarse grid.  Y[2*dim] is
+  the color matrix that is diagonal on the coarse
+  grid
+  */
```

Comment:
Doxygen and move to header

Reply:
See above

Reply:
Again, make sure all details are in header and delete from the .cpp file.  Can put in a in line or two of comments if warranted, but details like this should be in header.

Reply:
Will fix soon

Reply:
done